test/shared/markdown-serializer.test.ts

```diff
@@ -78,6 +78,10 @@ describe("markdown-serializer", () => {
             `<a href="https://www.example.com">https://www.example.com</a>`,
             `<https://www.example.com>`,
         ],
+        [
+            "<strong>Bold </strong><em><strong>Both </strong></em><em>Italic</em>",
```

Comment:
The html generated by the editor is:

```html
<p><strong>This is bold, </strong><em><strong>this is both</strong>, this is emphasis</em></p>
```

Note the em wrapping both. Ideally, we should unit test this in a more controlled manner by using the exact doc that is generated in this state:

```json
{
    "type": "doc",
    "attrs": {
        "markup": ""
    },
    "content": [
        {
            "type": "paragraph",
            "attrs": {
                "markup": ""
            },
            "content": [
                {
                    "type": "text",
                    "marks": [
                        {
                            "type": "strong",
                            "attrs": {
                                "markup": ""
                            }
                        }
                    ],
                    "text": "This is bold, "
                },
                {
                    "type": "text",
                    "marks": [
                        {
                            "type": "em",
                            "attrs": {
                                "markup": ""
                            }
                        },
                        {
                            "type": "strong",
                            "attrs": {
                                "markup": ""
                            }
                        }
                    ],
                    "text": "this is both"
                },
                {
                    "type": "text",
                    "marks": [
                        {
                            "type": "em",
                            "attrs": {
                                "markup": ""
                            }
                        }
                    ],
                    "text": ", this is emphasis"
                }
            ]
        }
    ]
}
```